apps/fl/style/download.css

```diff
@@ -26,9 +26,12 @@ body {
   font-size: 1.8rem;
 }
 
-#download-progress {
+#download-progress #direct-download-progress {
```

Comment:
You need a comma between these two.  As it stands, these styles indicate that the second element is nested inside of the first.  From the screenshot, it looks like the code is working okay, but I suspect that this change fixes the direct download bar but breaks the other progress bar.